tests/auth.setup.ts

```diff
@@ -0,0 +1,38 @@
+import { expect, test as setup } from '@playwright/test'
```

Comment:
this can bring confusion since it's well spread by the node community to have a `test` key

when I saw `setup` on the code, I went back to the imports to see if it was a method we made and not from the test library. this likely will bring confusion to contributors

```suggestion
import { expect, test } from '@playwright/test'
```

Reply:
Again, I have used the [official tutorial(https://playwright.dev/docs/auth) and that is how it has been done there